minigalaxy/ui/gametile.py

```diff
@@ -544,7 +547,7 @@ def __state_installable(self):
             self.progress_bar.destroy()
 
     def __state_queued(self):
-        self.button.set_label(_("in queue…"))
+        self.button.set_label(_("In queue…"))
```

Comment:
This change makes it necessary to update all translation files. So we'd either have to leave this change out or update those.